src/Components/Body.tsx

```diff
@@ -78,18 +86,75 @@ export function Body(props: State) {
   }
 
   const handleClick = (key, value) => {
+    var dateContent = props.dateTextContents;
+    var words = dateContent[props.index].split(" ");
+    if (key == "day") {

```

Comment:
Use an enum, if this is what I think it is.

Reply:
Ok. And I’ll do this for every function